### PR TITLE
Ops 1420/task 1520 maps contracts

### PR DESCRIPTION
**Pull request by johndeange**
## What changed

Remove `number` field from Agreement.
Add `local-migration` env and Dockerfile for MAPS/OPS testing.
Fixes for tests in new FY.

## Issue

https://github.com/HHS/OPRE-OPS/issues/1520

## Definition of Done Checklist
- ~~[ ] OESA: Code refactored for clarity~~
- [x] OESA: Dependency rules followed
- [x] Automated unit tests updated and passed
- [x] Automated integration tests updated and passed
- [x] Automated quality tests updated and passed
- [x] Automated load tests updated and passed
- [x] Automated a11y tests updated and passed
- [x] Automated security tests updated and passed
- [x] 90%+ Code coverage achieved
- ~~[ ] Form validations updated~~

### CODE REVIEW

**Comment by jonnalley** [comment]
@johndeange what issues do we create by removing the agreement number?   any issues with normalization ?

**Comment by johndeange** [comment]
> @johndeange what issues do we create by removing the agreement number? any issues with normalization ?

None that I am aware of - it doesn't seem like we were doing much with it.  A Contract has a `contract_number` but other Agreement types may not have a number.

File: frontend/cypress/e2e/loginPage.cy.js [hunk 1]
```diff
@@ -1,9 +1,15 @@
 /// <reference types="cypress" />
 import { testLogin } from "./utils";
+import { getCurrentFiscalYear } from "../../src/helpers/utils";
 
 it("has expected state on initial load", () => {
     cy.visit("/login");
     cy.fixture("initial-state").then((initState) => {
+        const currentFY = getCurrentFiscalYear();
```
Comment (fpigeonjr) [inline frontend/cypress/e2e/loginPage.cy.js:8]: 🥇

File: frontend/cypress/e2e/portfolioDetail.cy.js [hunk 2]
```diff
@@ -4,6 +4,7 @@ import { terminalLog, testLogin } from "./utils";
 beforeEach(() => {
     testLogin("admin");
     cy.visit("/portfolios/1");
+    cy.get("#fiscal-year-select").select("2023");
```
Comment (fpigeonjr) [inline frontend/cypress/e2e/portfolioDetail.cy.js:7]: 👍🏿

**Review by fpigeonjr — APPROVED** [review]

**Pull request merged**